### PR TITLE
fix: twitter data

### PR DESCRIPTION
**Pull request by juliusmarminge**
created a new token and added it to vercel

setting cache control to public to further reduce requests, maybe that helps?

### CODE REVIEW

File: www/src/components/landingPage/tweets/tweets.astro [hunk 1]
```diff
@@ -60,11 +60,8 @@ if (import.meta.env.MODE === "CI" || !token) {
     "1570054715240763393",
   ]);
 }
-const SIX_HOURS_IN_SECONDS = 6 * 60 * 60;
-Astro.response.headers.set(
-  "Cache-Control",
-  `s-max-age=${SIX_HOURS_IN_SECONDS}`,
-);
+const THREE_HOURS = 3 * 60 * 60;
```
Comment (nexxeln) [inline www/src/components/landingPage/tweets/tweets.astro:63]: Why did we make it 3 hours from 6 hours?

Comment (juliusmarminge) [reply www/src/components/landingPage/tweets/tweets.astro:63]: Idk just felt like it ;)

Public cache so will still reduce requests quite a bit

Comment (c-ehrlich) [reply www/src/components/landingPage/tweets/tweets.astro:63]: 3 hours seems fine to me, should not get rate limited on that

Comment (juliusmarminge) [reply www/src/components/landingPage/tweets/tweets.astro:63]: I have elevated access on my twitter app which gives me 2M requests per month, should be good

**Review by c-ehrlich — APPROVED** [review]

**Pull request merged**